lms/djangoapps/teams/views.py

```diff
@@ -1055,7 +1055,7 @@ def get(self, request):
                     CourseAccessRole.objects.filter(user=request.user, role='staff').values_list('course_id', flat=True)
                 )
                 accessible_course_ids = [item for sublist in (enrolled_courses, staff_courses) for item in sublist]
```

Comment:
I've been staring at that list comprehension for a little while and still don't really understand it...

Reply:
It's supposedly equivalent to:

```python
for sublist in (enrolled_courses, staff_courses):
    for item in sublist:
        course_ids.append(item)
```